content/index.md

```diff
@@ -17,70 +17,275 @@ footer:
 {% include box.html type="start" title="Summary" class="" %}
 {:/}
 
-This page contains list of ACT Rules to test conformance [Web Content Accessibility Guidelines (WCAG) {% include_cached icon.html name="different-view" %}](https://www.w3.org/WAI/standards-guidelines/wcag/), [WAI-ARIA {% include_cached icon.html name="different-view" %}](https://www.w3.org/WAI/ARIA/) and other accessibility practices.  ACT Rules for WCAG 2 are formally approved by the Accessibility Guidelines Working Group. Proposed rules have not yet completed the approval process. ACT Rules are informative — that means they are not required for determining conformance to WCAG or ARIA. Learn more from [About ACT Rules](/standards-guidelines/act/rules/about/).
+This page contains list of ACT Rules to test conformance [Web Content Accessibility Guidelines (WCAG) 2.2 {% include_cached icon.html name="different-view" %}](https://www.w3.org/WAI/standards-guidelines/wcag/), [WAI-ARIA {% include_cached icon.html name="different-view" %}](https://www.w3.org/WAI/ARIA/) and other accessibility practices.  ACT Rules are formally approved by the relevant Working Group. Proposed rules have not yet completed the approval process. ACT Rules are informative — that means they are not required for determining conformance to WCAG or ARIA. Learn more from [About ACT Rules](/standards-guidelines/act/rules/about/).
 
 {::nomarkdown}
 {% include box.html type="end" %}
 {:/}
 
```

Comment:
I think we should only have top-level headings in the ToC here. This would be "WCAG Rules" and "ARIA Rules" IMO.

Reply:
Resolving -- happy based on the discussions we've had about this.